doc/source/whatsnew/v0.20.3.txt

```diff
@@ -44,6 +44,7 @@ Conversion
 ^^^^^^^^^^
 
 - Bug in pickle compat prior to the v0.20.x series, when ``UTC`` is a timezone in a Series/DataFrame/Index (:issue:`16608`)
+- Bug in ``Series`` with ``dtype='category'`` (:issue:`16524`)
```

Comment:
be a bit more descriptive.

``Bug in Series construction when passing type=category``